scripts/country-uk/data.py

```diff
@@ -0,0 +1,89 @@
+# The UK Parliament (House of Commons) has members at consitituency level
+# in England, Wales, Scotland and Northern Ireland
+pcon = {
+    # data from: http://geoportal.statistics.gov.uk/datasets/westminster-parliamentary-constituencies-december-2017-full-extent-boundaries-in-the-uk-wgs84
+    'url': 'https://opendata.arcgis.com/datasets/5c582cef61d04618928639dd17e4f896_1.csv',
```

Comment:
Would it be better to commit the source CSV files here and document where they came from instead of linking them directly so that this process is more "reproducible". I can see reason for going either way on this. Happy to take guidance.